<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Post</class>
 <widget class="QWidget" name="Post">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="group_option">
     <property name="title">
      <string>Tuỳ chọn</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="ckb_post_to_groups">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Đăng bài vào nhóm</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Số nhóm muốn đăng:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sp_numposts">
          <property name="singleStep">
           <number>2</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Thể loại</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_group_type">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="txt_contents">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>300</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Nội dung theo định dạng {a | b c|...}</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="0">
         <widget class="QRadioButton" name="rb_image_enable">
          <property name="text">
           <string>Đăng ảnh</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QLineEdit" name="le_image_path">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QRadioButton" name="rb_status">
          <property name="text">
           <string>Đăng status</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QPushButton" name="btn_select_image">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Chọn Ảnh</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QCheckBox" name="chb_md5_enable">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Đổi MD5 image</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" colspan="3">
         <widget class="QCheckBox" name="chb_background_enable">
          <property name="text">
           <string>Dùng background</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QSpinBox" name="sp_image_count">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="value">
           <number>2</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_setting">
     <property name="title">
      <string>Cấu hình</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="rightMargin">
         <number>300</number>
        </property>
        <property name="horizontalSpacing">
         <number>6</number>
        </property>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_2">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Thời gian nghỉ(s):</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QSpinBox" name="sp_idle_from">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_6">
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>đến</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QSpinBox" name="sp_idle_to">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>60</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Số luồng:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="sp_threads">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="btn_schedule">
          <property name="styleSheet">
           <string notr="true">    background-color: purple;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    padding: 6px;
	color: white</string>
          </property>
          <property name="text">
           <string>Lập lịch</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_runnow">
          <property name="styleSheet">
           <string notr="true">    background-color: green;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    padding: 6px;
	color: white</string>
          </property>
          <property name="text">
           <string>Chạy ngay</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_cancel">
          <property name="styleSheet">
           <string notr="true">    background-color: red;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    padding: 6px;
	color: white</string>
          </property>
          <property name="text">
           <string>Huỷ</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rb_status</sender>
   <signal>toggled(bool)</signal>
   <receiver>chb_background_enable</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rb_image_enable</sender>
   <signal>toggled(bool)</signal>
   <receiver>chb_md5_enable</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rb_image_enable</sender>
   <signal>toggled(bool)</signal>
   <receiver>le_image_path</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rb_image_enable</sender>
   <signal>toggled(bool)</signal>
   <receiver>btn_select_image</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rb_image_enable</sender>
   <signal>toggled(bool)</signal>
   <receiver>sp_image_count</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_cancel</sender>
   <signal>clicked()</signal>
   <receiver>Post</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>405</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>433</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
